woocommerce/woocommerce
# Delete order data from wc_order_tax_lookup and wc_order_stats on order deletion

### All Submissions:


### Changes proposed in this Pull Request:

When the orders table is authoritative, deleting an order was not deleting the records in `wp_wc_order_tax_lookup` and `wp_wc_order_stats` tables for the order. This pull request fixes that.

Closes #36578.


### How to test the changes in this Pull Request:

1. Set the orders table as authoritative (Settings - Advanced - Custom data stores). It doesn't matter if sync is enabled or not (you can try both cases).
2. Create an order that has line items with taxes.
3. Verify that records exist for the order in the `wp_wc_order_tax_lookup` and `wp_wc_order_stats` tables.
4. Trash the order, then delete it.
5. Verify that the order has been deleted from `wp_wc_orders` **and** the records in `wp_wc_order_tax_lookup` and `wp_wc_order_stats` for the order have been deleted as well.

### Other information:


### FOR PR REVIEWER ONLY:
